src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -646,7 +646,12 @@ public function dumpFile($filename, $content)
             throw new IOException(sprintf('Failed to write file "%s".', $filename), 0, null, $filename);
         }
 
-        @chmod($tmpFile, 0666 & ~umask());
```

Comment:
`$this->chmod` wouldn't provide much more to me, except the exception on error, but I don't see it as needed here